pages/mfbarrera/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Hotels</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <section>
+    <div class="container">
+      <div class="background">
+        <div class="carousel__header">OUR HOTELS</div>
+        <div class="carousel">
+          <button class="button button--left">
+            <img class="button__image"
+              src="https://cdn.iconscout.com/icon/premium/png-512-thumb/left-caret-1578336-1341870.png">
+          </button>
```

Comment:
Do you think the `button class` represents how this element looks? I mean, it is an arrow, not a button.

Reply:
I think it is a button that can be used in other components.